### DISCUSSION TITLE
Beta 1.0!

### DISCUSSION THREAD

**Discussion by gusmanb** [gusmanb/ZXBasicStudio#23] [Announcements]
This is the first beta release of the application. Expect bugs as this is a work in progress :)

To report any problem with it please use the issues section of the repository.

Have fun!

<hr /><em>This discussion was created from the release <a href='https://github.com/gusmanb/ZXBasicStudio/releases/tag/v1.0-beta.1'>Beta 1.0!</a>.</em>

**Comment by Andy1966uk** [comment]
Thanks for sharing, works well
